rotkehlchen/globaldb/handler.py

```diff
@@ -1055,6 +1055,28 @@ def get_manual_current_price(asset: Asset) -> Optional[Tuple[Asset, Price]]:
 
         return Asset(result[0]), deserialize_price(result[1])
 
+    @staticmethod
+    def get_all_manual_current_prices(
+            from_asset: Optional[Asset] = None,
+            to_asset: Optional[Asset] = None,
+    ) -> List[Tuple[Asset, Asset, Price]]:
+        """Returns all the manual current prices in the price_history table"""
+        query = 'SELECT from_asset, to_asset, price FROM price_history WHERE source_type=?'
+        bindings = [HistoricalPriceOracle.MANUAL_CURRENT.serialize_for_db()]
+        if from_asset is not None:
+            query += ' AND from_asset=?'
+            bindings.append(from_asset.identifier)
+        if to_asset is not None:
+            query += ' AND to_asset=?'
+            bindings.append(to_asset.identifier)
+
+        with GlobalDBHandler().conn.read_ctx() as read_cursor:
+            result = read_cursor.execute(query, bindings)
+            return [
+                (Asset(entry[0]), Asset(entry[1]), deserialize_price(entry[2]))
```

Comment:
Do you really need to deserialize to `Asset` here? Isn't this result just going out of the API, which means it just gets turned to an identifer again?

Reply:
I made it follow the pattern in the handler of deserializing when is needed but yes... is getting the asset to only return the identifier later. If is okey with you will remove that